tools/scaffolder/cmd/main.go

```diff
@@ -9,31 +9,31 @@ import (
 )
 
 var (
-	configPath string
-	crdKind    string
-	listCRDs   bool
+	inputPath string
+	crdKind   string
+	listCRDs  bool
 )
 
 func main() {
 	rootCmd := &cobra.Command{
 		Use:   "ako-controller-scaffolder",
 		Short: "Generate Kubernetes controllers for MongoDB Atlas CRDs",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if configPath == "" {
-				return fmt.Errorf("--config is required")
+			if inputPath == "" {
+				return fmt.Errorf("--input is required")
 			}
 
 			if listCRDs {
-				return generate.PrintCRDs(configPath)
+				return generate.PrintCRDs(inputPath)
 			}
 
-			return generate.FromConfig(configPath, crdKind)
+			return generate.FromConfig(inputPath, crdKind)
 		},
 	}
 
-	rootCmd.Flags().StringVar(&configPath, "config", "", "Path to atlas2crd config file (required)")
+	rootCmd.Flags().StringVar(&inputPath, "input", "", "Path to openapi2crd result.yaml file (required)")
 	rootCmd.Flags().StringVar(&crdKind, "crd", "", "CRD kind to generate controller for")
-	rootCmd.Flags().BoolVar(&listCRDs, "list", false, "List available CRDs from config file")
+	rootCmd.Flags().BoolVar(&listCRDs, "list", false, "List available CRDs from result file")
```

Comment:
I don't understand what this option is doing. What does "Listing availabile from CRDs" mean? What "result" file?

Reply:
This one lists available CRDs to generate stuff for, if the CRD is a multi-document yaml.